Cargo.toml

```diff
@@ -149,6 +149,12 @@ rust.single_use_lifetimes = "warn"
 rust.unused_lifetimes = "warn"
 # TODO: reenable
 # rust.unsafe_op_in_unsafe_fn = "deny"
+rust.unexpected_cfgs = { level = "warn", check-cfg = [
+    # FIXME: https://github.com/hyperledger/iroha/issues/3102
+    'cfg(feature, values("ffi_import"))',
+    # FIXME: was never supported
+    'cfg(coverage)'
```

Comment:
how about you make a ticket?

Reply:
https://github.com/hyperledger/iroha/issues/5139

Reply:
Actually it is not a bug (https://github.com/hyperledger/iroha/issues/5139#issuecomment-2401284049). I will leave a better comment here.